src/mesh/unstructured/unstructuredMesh.cpp

```diff
@@ -105,4 +105,105 @@ UnstructuredMesh createSingleCellMesh(const Executor exec)
         boundaryMesh
     );
 }
+
+/** @brief A factory function for a 1D mesh
+ *
+ * A 1D mesh in 3D space in which each cell has a left and a right face.
+ * The 1D mesh is aligned with the x coordinate of Cartesian coordinate system.
+ */
+UnstructuredMesh create1DUniformMesh(const Executor exec, const size_t nCells)
```

Comment:
The docstring should be in the header file `unstructuredMesh.hpp`
```suggestion
UnstructuredMesh create1DUniformMesh(const Executor exec, const size_t nCells)
```

Reply:
This docstring should be removed